<?xml version="1.0" encoding="UTF-8"?>
<launch>
    <arg name="rviz" default="true"/>
    <rosparam command="load" file="$(find fabrics_bridge)/config/fabrics_panda_minimal_config.yaml"/>
    <group ns="fabrics">
        <!-- start fabric node -->
        <node pkg="fabrics_bridge" type="panda_fabrics_node" name="panda_fabrics_node" output="screen"/>
        <node pkg="fabrics_bridge" type="panda_helpers_node" name="panda_helpers_node" output="screen"/>
    </group>
    <node pkg="rviz" type="rviz" name="rviz" args="-d $(find fabrics_bridge)/config/fabrics_panda_visualization.rviz" if="$(arg rviz)"/>

</launch>
